agent/Agent.js

```diff
@@ -160,6 +162,9 @@ class Agent extends EventEmitter {
     bridge.on('selected', id => this.emit('selected', id));
     bridge.on('setInspectEnabled', enabled => {
       this._inspectEnabled = enabled;
+      if (enabled) {
+        this.emit('stopInspecting');
```

Comment:
You're right, it wasn't necessary. I think it was zombie code from when I was debugging. Oops!